fontforge/parsettfatt.c

```diff
@@ -776,6 +776,14 @@ static AnchorClass **MarkGlyphsProcessMarks(FILE *ttf,int markoffset,
     struct mr { uint16 class, offset; } *at_offsets;
     SplineChar *sc;
 
+    fseek(ttf,markoffset,SEEK_SET);
```

Comment:
Hi, it's to avoid potential useless chunk alloc in "for ( i=0; i<classcnt; ++i )" loop